Meadowlark-js/packages/meadowlark-core/test/handler/DeleteIt.test.ts

```diff
@@ -52,6 +52,41 @@ describe('given there is no resourceId', () => {
   });
 });
 
+describe('given delete has write conflict failure', () => {
+  let response: FrontendResponse;
+  let mockDocumentStore: any;
+
+  beforeAll(async () => {
+    mockDocumentStore = jest.spyOn(PluginLoader, 'getDocumentStore').mockReturnValue({
+      ...NoDocumentStorePlugin,
+      deleteDocumentById: async () =>
+        Promise.resolve({
+          response: 'DELETE_FAILURE_WRITE_CONFLICT',
+          failureMessage: 'Write conflict error returned',
```

Comment:
While this message is certainly true, I think there could be a better API client-facing message for these 409s. There should be an explanation why. Something like "Write conflict due to concurrent access to this or related resources."